src/components/Background/index.css

```diff
@@ -55,11 +55,14 @@
 
 .video {
   position: absolute !important;
-  top: -50%;
-  left: -50%;
+  top: 0px;
+  top: 0px;
 
-  width: 200%;
-  height: 200%;
+  width: auto;
+  height: auto;
+
+  min-width: 100%;
```

Comment:
Don't forget to check on IE 11

Reply:
The video shows up right on IE, just for a second and 💣 (also on svhealth.io)

Reply:
Dude, aren't you supposed to be on vacation? 😆

Reply:
Just took a couple minutes off of my "off time" 😂

Reply:
Well, I create the PR #92, but IE continue to 💣. But we talk about this next year 😁